src/content/service.js

```diff
@@ -199,10 +199,12 @@ module.exports = async ({ botfile, projectLocation, logger, ghostManager }) => {
   }
 
   const categoryItemsCount = () => {
-    return knex('content_items')
-      .count('id as count')
-      .get(0)
-      .get('count')
+    return Number(
```

Comment:
Oh, this is wrong, it converts the _promise_ to Number. Should either be `Number(await ...)` (and async in function declaration) or `.then(Number)` in the end of the chain

Reply:
Sorry, fixed that.